bedrock/settings/base.py

```diff
@@ -24,29 +26,31 @@ def path(*args):
 
 
 # Is this a dev instance?
-DEV = False
+DEV = config('DEV', cast=bool, default=False)
 
-DEBUG = False
-TEMPLATE_DEBUG = DEBUG
+DEBUG = TEMPLATE_DEBUG = config('DEBUG', cast=bool, default=False)
 
 # Production uses MySQL, but Sqlite should be sufficient for local development.
 # Our CI server tests against MySQL.
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': 'bedrock.db',
-    }
-}
-SLAVE_DATABASES = []
+DATABASES = config(
```

Comment:
The other strategy I was envisioning was an optional env var as a CSV of other env var names. e.g.:

```python
DATABASES = {}
DATABASE_URL_VARS = config('DATABASE_URL_VARS', default=None, cast=Csv())
if DATABASE_URL_VARS:
    # example usage
    # DATABASE_URL_VARS='DATABASE_URL_DEFAULT,DATABASE_URL_RW_MASTER'
    # DATABASE_URL_DEFAULT='sqlite:///bedrock.db'
    # DATABASE_URL_RW_MASTER='sqlite:///rw_master.db'
    for db_var in DATABASE_URL_VARS:
        db_name = db_var[13:].lower()
        DATABASES[db_name] = dj_database_url.parse(config(db_var))
else:
    DATABASES['default'] = config('DATABASE_URL', default='sqlite:///bedrock.db')
```

I don't think the above is as elegant a solution as yours, but it sure would make the environment vars and `.env` file easier to read.